test/dlc_tests/conftest.py

```diff
@@ -644,9 +647,13 @@ def delete_ssh_keypair():
         )
         or (
             "tensorflow_inference" in request.fixturenames
-            and "graviton_compatible_only" in request.fixturenames
+            and (
```

Comment:
do we know why this condition only exists under TF inference? curious if it is needed anymore

Reply:
no I'm not sure, but yeah the logic does seem redundant. We could rethink this whole giant if-statement tbh